webpki-root-certs/tests/codegen.rs

```diff
@@ -0,0 +1,82 @@
+use std::fmt::Write;
```

Comment:
Is it worth having a single codegen test that writes both of the crates?

Reply:
It didn't feel to me like there was enough shared logic to make it worthwhile.